tests/helpers.py

```diff
@@ -2056,3 +2057,200 @@ def modify_osd_replica_count(resource_name, replica_count):
     params = f'{{"spec": {{"replicas": {replica_count}}}}}'
     resource_name = '-'.join(resource_name.split('-')[0:4])
     return ocp_obj.patch(resource_name=resource_name, params=params)
+
+
+def check_required_osd_count(total_osd_nos=3):
```

Comment:
can you rename the function to `add_n_osds` or something similar? You are doing more than `check` here.

Reply:
ack